libs/filamat/src/PostprocessMaterialBuilder.cpp

```diff
@@ -40,6 +40,12 @@ using namespace filament::backend;
 
 namespace filamat {
 
+inline void assertSingleTargetApi(MaterialBuilderBase::TargetApi api) {
```

Comment:
This is duplicated here, but `PostprocessMaterialBuilder` will be deprecated soon anyway